includes/dashboard.php

```diff
@@ -797,7 +797,7 @@ function action_admin_enqueue_dashboard_scripts() {
 		$data = array( 'nonce' => wp_create_nonce( 'ep_dashboard_nonce' ) );
 
 		$index_meta = \ElasticPress\Utils\get_indexing_status();
-		$wpcli_sync = Utils\is_indexing_wpcli();
+		$wpcli_sync           = (bool) get_transient( 'ep_wpcli_sync' );
```

Comment:
I think this whitespace change should be this
```suggestion
		$wpcli_sync = (bool) get_transient( 'ep_wpcli_sync' );
```